# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/gzip/gzip-1.3.5-zforce.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- gzip-1.2.4/zforce.in.sopwith	Thu Feb  3 18:06:30 2000
+++ gzip-1.2.4/zforce.in	Thu Feb  3 18:07:37 2000
@@ -25,7 +25,7 @@
   test `expr "$i" : '.*[.-]gz$'` -eq 0 || continue
   test `expr "$i" : '.*[.]t[ag]z$'` -eq 0 || continue
 
-  if gzip -lv < "$i" 2>/dev/null | grep '^defl' > /dev/null; then
+  if gzip -l < "$i" 2>/dev/null | grep '^compressed' > /dev/null; then
 
     if test `expr "$i" : '^............'` -eq 12; then
       new=`expr "$i" : '\(.*\)...$`.gz
